src/vms/pvm/txs/fee/constants.ts

```diff
@@ -0,0 +1,206 @@
+import type { Dimensions } from '../../../common/fees/dimensions';
+import { FeeDimensions } from '../../../common/fees/dimensions';
+import {
+  PUBLIC_KEY_LENGTH,
+  SIGNATURE_LENGTH as BLS_SIGNATURE_LENGTH,
+} from '../../../../crypto/bls';
+import { SIGNATURE_LENGTH } from '../../../../crypto/secp256k1';
+
+/**
+ * Number of bytes per long.
+ */
+const LONG_LEN = 8;
+
+export const ID_LEN = 32;
+
+/**
+ * Number of bytes per short.
+ */
+const SHORT_LEN = 2;
+
+export const SHORT_ID_LEN = 20;
+
+/**
+ * Number of bytes per int.
+ */
+const INT_LEN = 4;
```

Comment:
Maybe there is a better place to move these constants since they aren't specific to complexity? I couldn't find any existing constants but maybe I missed something?